students/danwoj/Session12/trapz.py

```diff
@@ -0,0 +1,24 @@
+def quadratic(x, A=0, B=0, C=0):
+    return A * x**2 + B * x + C
+
+def trapz(fun, a, b, *args, **kwargs):
+    '''
+    Trapezoidal Function
+    '''
+
+    # The next four lines are generating 100 evenly distributed 
+    # x coordinates between points a and b
+    x_values = []
+    x_values.append(a+((b-a)/100))
+    for i in range(1,101):
+        x_values.append(x_values[i-1]+((b-a)/100))
```

Comment:
It's a little better to calculate the value at each point:

``a + i*(b-a)/100``

rather than adding them up as you go -- each value is not exact, so the error accumulate.